tests/integration-tests/Android/Android.CLI.Commands.Tests.proj

```diff
@@ -0,0 +1,80 @@
+    <HelixType>test/product/</HelixType>
+    <IncludeXHarnessCli>true</IncludeXHarnessCli>
+    <TestRunNamePrefix>$(AGENT_JOBNAME)</TestRunNamePrefix>
+    <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
+    <HelixBaseUri>https://helix.dot.net</HelixBaseUri>
+    <!-- Pick up the nupkg from this repo for testing purposes -->
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-ci</MicrosoftDotNetXHarnessCLIVersion>
+    <XHarnessNupkgPath>$(MSBuildThisFileDirectory)../../../artifacts/packages/$(Configuration)/Shipping/Microsoft.DotNet.XHarness.CLI.$(MicrosoftDotNetXHarnessCLIVersion).nupkg</XHarnessNupkgPath>
+
+    <!-- Required: Version of XHarness CLI to use -->
+    <IncludeXHarnessCli>true</IncludeXHarnessCli>
+
+    <!-- Optional: Properties that are also valid for the Arcade Helix SDK (some might be needed for CI runs only) -->
+    <HelixType>test/product/</HelixType>
+    <HelixBaseUri>https://helix.int-dot.net</HelixBaseUri>
+    <Creator>$(BUILD_SOURCEVERSIONAUTHOR)</Creator>
+    <EnableXUnitReporter>true</EnableXUnitReporter>
+    <EnableAzurePipelinesReporter>true</EnableAzurePipelinesReporter>
+  </PropertyGroup>
+
+  <!-- For non-ci local runs -->
+  <PropertyGroup Condition=" '$(AGENT_JOBNAME)' == '' ">
+    <EnableAzurePipelinesReporter>false</EnableAzurePipelinesReporter>
+    <MicrosoftDotNetXHarnessCLIVersion>1.0.0-dev</MicrosoftDotNetXHarnessCLIVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <XHarnessNupkgPath>$(ArtifactsShippingPackagesDir)/Microsoft.DotNet.XHarness.CLI.$(MicrosoftDotNetXHarnessCLIVersion).nupkg</XHarnessNupkgPath>
+  </PropertyGroup>
+
+  <!-- Required: Configuration that is already needed for the Helix SDK -->
+  <ItemGroup>
+    <HelixTargetQueue Include="ubuntu.1804.amd64.android.open"/>
+  </ItemGroup>
+
+  <PropertyGroup Condition=" '$(HelixAccessToken)' == '' ">
+    <IsExternal>true</IsExternal>
+    <Creator>$(BUILD_SOURCEVERSIONAUTHOR)</Creator>
+    <Creator Condition=" '$(Creator)' == '' ">anon</Creator>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <XHarnessX86TestApkUrl>https://netcorenativeassets.blob.core.windows.net/resource-packages/external/android/test-apk/x86/System.Numerics.Vectors.Tests-x86.zip</XHarnessX86TestApkUrl>
+  </PropertyGroup>
+
+  <!-- Useless stuff to make Arcade SDK happy -->
+  <PropertyGroup>
+    <Language>msbuild</Language>
+  </PropertyGroup>
+
+  <Target Name="TestAndroid" BeforeTargets="CoreTest">
+
+    <DownloadFile SourceUrl="$(XHarnessX86TestApkUrl)" DestinationFolder="$(ArtifactsTmpDir)apk" SkipUnchangedFiles="True" Retries="5">
+      <Output TaskParameter="DownloadedFile" ItemName="DownloadedApkFile" />
+    </DownloadFile>
+
+    <Message Text="Downloaded @(DownloadedApkFile) for XHarness Test purposes" Importance="High" />
+
+    <ItemGroup>
+      <HelixWorkItem Include="System.Numerics.Vectors.Tests">
+        <Command>
+          deviceId=`dotnet exec $XHARNESS_CLI_PATH android device --output-directory=out | head -2 | tail -1`;
```

Comment:
Looks like the build is failing on the `--output-directory` arg:
https://helixre8s23ayy6tv7v8isja.blob.core.windows.net/dotnet-xharness-refs-pull-503-merge-91e0612ca03e45dab2/System.Numerics.Vectors.Tests/console.d7b13d44.log?sv=2019-07-07&se=2021-04-21T09%3A09%3A18Z&sr=c&sp=rl&sig=%2BQnvtDX2cK00JStirtd1y4jHjr0Hq5y5sBI3xRm28OU%3D

This should fix it:
```suggestion
          set -e;
          deviceId=`dotnet exec "$XHARNESS_CLI_PATH" android device | head -2 | tail -1`;
```

(also add `-e` to fail when some commands fail)